src/components/date-time-input/date-time-input.ts

```diff
@@ -339,11 +338,13 @@ export default class IgcDateTimeInputComponent extends EventEmitterMixin<
 
     addKeybindings(this, {
       skip: () => this.readOnly,
-      bindingDefaults: { preventDefault: true },
+      bindingDefaults: { preventDefault: true, triggers: ['keydownRepeat'] },
     })
-      .set([ctrlKey, ';'], () => {
-        this.value = new Date();
-      })
+      // Skip default spin when in the context of a date picker
+      .set([altKey, arrowUp], noop)
+      .set([altKey, arrowDown], noop)
```

Comment:
I'm assuming we'll need to re-emit the keydown on our own to be able to react to default prevented and skip performing the action? Ideally, would love to have the date picked handling localized in that component, but I understand this small compromise. Just something to consider for the future if more of those cases arise.

Reply:
Kind of similar w/ the calendar styling exclusion in the dialog, though that's reasonable enough.